segment-appcues/src/main/java/com/appcues/segment/AppcuesDestination.kt

```diff
@@ -40,29 +41,44 @@ class AppcuesDestination(
 
     var appcues: Appcues? = null
 
+    @SuppressWarnings("TooGenericExceptionCaught")
     override fun update(settings: Settings, type: Plugin.UpdateType) {
         super.update(settings, type)
 
         if (settings.hasIntegrationSettings(this)) {
```

Comment:
Does it make sense to move the 
```
if (type == Plugin.UpdateType.Initial)
```
to be the outer if statement? Do nothing unless that is true? I'm thinking we may not need to do line 48 at all, if that check for `Initial` is false, right?

Reply:
I thought that was weird too, but decide to follow the other [impls](https://github.com/braze-inc/analytics-kotlin-braze/blob/main/lib/src/main/java/com/segment/analytics/kotlin/destinations/braze/BrazeDestination.kt#L49)

Reply:
ok, fair enough - I don't think it would make a huge change either way probably.